modules/dateTime.js

```diff
@@ -0,0 +1,9 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+const currentTime = document.querySelector('#currentTime');
```

Comment:
- [ ] kindly display the date and time on the page
<img width="1440" alt="Screen Shot 2022-08-08 at 10 55 56" src="https://user-images.githubusercontent.com/81444338/183483564-b3314d61-3671-49e2-b04d-248dbea5c4e5.png">

Reply:
![giphy (3)](https://user-images.githubusercontent.com/105079888/183494483-85708cac-ee00-4d9c-88f1-87f9664d34f9.gif)